website/docs/commands/init.html.markdown

```diff
@@ -116,37 +116,41 @@ initialized with its child modules.
 
 ## Plugin Installation
 
-During init, the configuration is searched for both direct and indirect
-references to [providers](/docs/configuration/providers.html), and the plugins
-for the providers are retrieved from the plugin repository. The downloaded
-plugins are installed to a subdirectory of the working directory, and are thus
-local to that working directory.
-
-Re-running init with plugins already installed will install plugins only for
-any providers that were added to the configuration since the last init. Use
-`-upgrade` to additionally update already-installed plugins to the latest
-versions that comply with the version constraints given in configuration.
-
-To skip plugin installation, use `-get-plugins=false`.
-
-The automatic plugin installation behavior can be overridden by extracting
-the desired providers into a local directory and using the additional option
-`-plugin-dir=PATH`. When this option is specified, _only_ the given directory
-is consulted, which prevents Terraform from making requests to the plugin
-repository or looking for plugins in other local directories. Passing an empty
-string to `-plugin-dir` removes any previously recorded paths.
-
-Custom plugins can be used along with automatically installed plugins by
-placing them in `terraform.d/plugins/OS_ARCH/` inside the directory being
-initialized. Plugins found here will take precedence if they meet the required
-constraints in the configuration. The `init` command will continue to
-automatically download other plugins as needed.
-
-When plugins are automatically downloaded and installed, by default the
-contents are verified against an official HashiCorp release signature to
-ensure that they were not corrupted or tampered with during download. It is
-recommended to allow Terraform to make these checks, but if desired they may
-be disabled using the option `-verify-plugins=false`.
+During init, Terraform searches the configuration for both direct and indirect
+references to providers and attempts to load the required plugins.
+
+For [providers distributed by HashiCorp](/docs/providers/index.html),
+init will automatically download and install plugins if necessary. Other plugins
+can be manually installed in the user plugins directory, which is located at
+`~/.terraform.d/plugins` on most operating systems and
+`<APPLICATION DATA>\plugins` on Windows.
+
+For more information about configuring and installing providers, see
+[Configuration: Providers](/docs/configuration/providers.html).
+
+On subsequent runs, init will keep any previously installed plugins and only
+install new providers that were added to the configuration since the last init.
```

Comment:
I'm not sure if it's worth mentioning here, but if the provider version constraint changes, and the existing version no long meets the constraints, terraform init will pull the correct version (be that up- or down-grade). 

I don't have a good sense if that's too much info for this block or not so this shouldn't keep you from merging.